src/dynapi/SDL_dynapi.c

```diff
@@ -89,6 +89,12 @@ static void SDL_InitDynamicAPI(void);
         va_end(ap); \
         return retval; \
     } \
+    _static int SDLCALL SDL_asprintf##name(char **strp, SDL_PRINTF_FORMAT_STRING const char *fmt, ...) { \
```

Comment:
I don't know whether this is the correct place (order) for new varargs procs, but should it not be at end?

Reply:
The `SDL_DYNAPI_VARARGS` macro isn't used directly inside the jump tables, so I don't think the order is critically important.